lib/active_triples/data_model.rb

```diff
@@ -0,0 +1,20 @@
+module ActiveTriples
+  ##
+  # Super class which provides a simple property DSL for defining property ->
+  # predicate mappings.
+  class DataModel
```

Comment:
PredicateMapping maybe, but the Property interface allows arbitrary metadata to be attached to a name (IE class_name).

Names are hard.